tests/test_scene_rendering/test_sections.py

```diff
@@ -80,3 +80,25 @@ def test_many_sections(tmp_path, manim_cfg_file, simple_scenes_path):
     ]
     _, err, exit_code = capture(command)
     assert exit_code == 0, err
+
+
+@pytest.mark.slow
+@video_comparison(
+    "SceneWithSkipAnimations.json",
+    "videos/simple_scenes/480p15/ElaborateSceneWithSections.mp4",
+)
+def test_skip_animations(tmp_path, manim_cfg_file, simple_scenes_path):
+    scene_name = "ElaborateSceneWithSections"
+    command = [
+        sys.executable,
+        "-m",
+        "manim",
+        "-ql",
+        "--save_sections",
+        "--media_dir",
+        str(tmp_path),
+        simple_scenes_path,
+        scene_name,
+    ]
+    _, err, exit_code = capture(command)
+    assert exit_code == 0, err
```

Comment:
I slightly dislike calling manim via its CLI when its not the CLI itself that is being tested. I have recently added a similar test where at least a dry-run of rendering needed to take place, and there I simply used 

```py
with tempconfig({'quality': 'low', ...}):
    s = MyScene()
    s.render()

...
```

Seeing that other tests here are written in the same way I'm fine with keeping this as it is for now, but for future tests I'd prefer simply calling the library from within the test. :-)

Reply:
This is deliberate as the testing you suggested—as far as I know, correct me if I'm wrong—uses a stub file writer. But the file output is part of what I'm testing here, when you skip the animations of a section, no output video file will be created. That's what I want to assert.

Reply:
AFAIK this would use the *real deal* and actually render + write the corresponding files. (I usually pass `"dry_run": True` for my purposes though; this wouldn't make sense here though.)

Reply:
Good to know.
Our framework for video testing has to be partly refactored anyways: #2207.